bzt/cli.py

```diff
@@ -86,22 +86,6 @@ def setup_logging(options):
             fmt_verbose = Formatter("[%(asctime)s %(levelname)s %(name)s] %(message)s")
             fmt_regular = Formatter("%(asctime)s %(levelname)s: %(message)s", "%H:%M:%S")
 
-        logger = logging.getLogger('')
-        logger.setLevel(logging.DEBUG)
-
-        # log everything to file
-        if options.log is None:
-            tf = tempfile.NamedTemporaryFile(prefix="bzt_", suffix=".log", delete=False)
-            tf.close()
-            os.chmod(tf.name, 0o644)
-            options.log = tf.name
-
-        if options.log:
```

Comment:
Old code was allowing to disable file logging, by providing empty string. New code does not allow that